testcase.m

```diff
@@ -0,0 +1,212 @@
+%% test case with Weick 1931 (pag 166 figura C-2)
+clear all; close all; clc
+V= 40; %m/s
+rpm= 4500; %giri al minuto
+R_dim= 1.58; %m
+D=1.58*2; %m
+n=rpm/60; %giri al secondo
+%R= 1
+r_hub_dim= 0.14; %m
+r_hub=r_hub_dim/R_dim;
+N= 3; %numero di pale
+ns= 10; %numero sezioni
+r= linspace(r_hub_dim, 0.97*R_dim, ns);
+r= r/1.58;
+R=R_dim/1.58;
+c= 0.01*[9, 15, 17, 18.5, 19, 19, 17.5, 15.5, 12.5, 9]; %m
+Cl_alpha= ones(1, ns);
+Cl_alpha= 6.28*Cl_alpha;
+beta_deg= [40, 39.7, 37.6, 32.3, 27.8, 24.4, 21.8, 19.9, 18.3, 17.6];
+x=linspace(r_hub,R,100);
+
+%omega= convangvel(rpm, 'rpm', 'rad/s')
+omega= rpm*2*pi/60;
+i=1;
+while i< ns+1
+beta(i)= convang(beta_deg(i), 'deg', 'rad');
+i= i+1;
+end
+
+%% Metodo A
+
+lam= V/(omega*R);
+sigma= N*c./(pi*r);
+Vt= omega*r;
+Vr= Vt.*sqrt((lam^2)+ (r.^2));
+phi= atan ((V)./(Vt));
+
+alpha_i_A= 0.5*(-(lam./r + sigma.*Cl_alpha./(8*r.^2).*(Vr./Vt)) +...
+    sqrt((lam./r + sigma.*Cl_alpha./(8*r.^2).* (Vr./Vt)).^2 +...
+    sigma.*Cl_alpha.*Vr.*(beta-phi)./(2*(r.^2).*Vt)));

```

Comment:
If you want to use non-dimensional variables it is not sufficient to just put them into the equation.
If you wanted to use dimensional variables, then check what's previously.